<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>account:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pass_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="login_btn">
       <property name="text">
        <string>login</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>151</width>
      <height>215</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>host:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="host_edit">
       <property name="text">
        <string>9.135.98.122</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="port_edit">
       <property name="text">
        <string>6789</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>bee</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bee_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="conn_btn">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconn_btn">
       <property name="text">
        <string>disconnect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="biu_btn">
       <property name="text">
        <string>biu biu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
